Python/sysmodule.c

```diff
@@ -1493,6 +1493,9 @@ static PyStructSequence_Desc windows_version_desc = {
 static PyObject *
 _sys_getwindowsversion_from_kernel32()
 {
+#ifdef MS_WINDOWS_NON_DESKTOP
```

Comment:
We no longer use the "kernel32.dll" file version anyway. It isn't 100% reliable. The platform module was updated to use a new `_wmi` module to get the real OS version.

Reply:
Is this something that needs changing, or just a comment on it not being really relevant?